app/views/comments/edit.js.erb

```diff
@@ -0,0 +1,11 @@
+$("#comment-content<%= @comment.id %>")
+  .html(
+    "<%= escape_javascript(render 'form', comment: @comment, display_comment_field: true )%>"
+  )
+  .find(".btn").after( "<small>   <%= escape_javascript(link_to 'Cancel', '#') %></small>" )
+  .next().find("a").on('click', function(e) {
+    e.preventDefault();
+    $(e.target).parents("#comment-content<%= @comment.id %>").html( "<%= escape_javascript(@comment.content) %>" );
+  });
+;
+
```

Comment:
This file is extremely difficult to read. Is there anything you can do to increase the readability of what you're doing here?